PROJECT_GOVERNANCE.md

```diff
@@ -0,0 +1,163 @@
+   community (with input from other maintainers), the decision can be
+   acted upon by those involved in the discussion.
+3. **Request Admin Intervention:** If possible, attempt to contact
+   admins through other channels (email, social media, etc.) to notify
+   them of the situation.
+4. **Temporary Action:** In the absence of admin response, maintainers
+   or active contributors may make temporary decisions, such as merging
+   a critical PR, until the admins return.
+
+**Note**: If an admin remains unresponsive, the other admins will
+continue to carry out their responsibilities and may make decisions on
+behalf of the project as necessary. If all admins become unresponsive,
+the community should follow the steps outlined above.
+
+**Note**: If maintainers or admins expect to be unavailable for an
+extended period, they are encouraged to notify the community in advance,
+if possible, to arrange for temporary coverage or decision-making
+processes.
+
+### Succession Planning
+
+To prevent prolonged unresponsiveness, it is advisable to periodically
+review the activity of admins and maintainers. If an admin becomes
+permanently inactive, the remaining admins may appoint a replacement
+from active community members. Similarly, inactive maintainers may be
+replaced or supplemented by other contributors with a track record of
+involvement in the project.
+
+## 2. Template-Specific Governance
+
+Each project within Tinted Theming is managed by a dedicated maintainer
+or maintainers. These maintainers are responsible for overseeing the
+development, updates, and quality of their respective templates.
+
+**Final Say on Templates:** Maintainers generally have the final say
+regarding their specific templates, as they possess the most intimate
+knowledge and expertise concerning their design and functionality.
+However, admins reserve the right to intervene if a template decision:
+
+- Significantly affects the broader project or community.
+- Conflicts with the project's core principles or guidelines.
+- Introduces potential security or usability concerns.
+
+In such cases, admins may discuss the issue with the template maintainer
+to reach a resolution, and if necessary, admins may have the final say.
+
+## 3. Acceptance Criteria for Pull Requests (PRs)
+
+To ensure the continued quality and consistency of the Tinted Theming
+project, all contributions are reviewed against the following acceptance
+criteria:
+
+### General PR Criteria:
+- **Alignment with Project Goals:** The PR should align with the overall
+  goals and philosophy of the Tinted Theming organization.
+- **Code Quality:** The code should be clean, well-documented, and
+  follow the project's coding standards.
+- **No Breaking Changes:** PRs should not introduce breaking changes
+  unless absolutely necessary. If a breaking change is required, it must
+  be well-documented, and a clear migration path should be provided.
```

Comment:
```suggestion
- **No Breaking Changes:** PRs should not introduce breaking changes to users
  unless absolutely necessary. If a breaking change is required, it must
  be well-documented, and a clear migration path should be provided.
```

Reply:
At least in my mind, it's good to remember we have far more users than maintainers - if we need to introduce a breaking change for our maintainers, we should still have a migration path, but it is slightly less important for it to be completely seamless. 

I'm open to being convinced otherwise though.